commcare_connect/opportunity/api/serializers.py

```diff
@@ -105,6 +108,8 @@ class OpportunitySerializer(serializers.ModelSerializer):
     payment_units = serializers.SerializerMethodField()
     is_user_suspended = serializers.SerializerMethodField()
     catchment_areas = serializers.SerializerMethodField()
+    start_time_threshold = serializers.SerializerMethodField()
```

Comment:
As an extension of my comment below, I am not sure this needs a method rather than just the `source` argument. Alternatively, I think this might be nicer in a new `verification_flags` object, rather than at the top level, since we are likely to add the others and it will help with organization.